app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+  @user = User.find(params[:id])
+  @user.update(user_params)
```

Comment:
- [ ] The same way you have the conditional at the create action, you'll need one here too. It should `redirect_to edit_user_path` if user was saved and render :edit otherwise.